client.lua

```diff
@@ -345,8 +346,23 @@ local function EnableTarget()
 				end
 			else sleep += 20 end
 			if not success then
-				local closestDis, closestZone
-				for _, zone in pairs(Zones) do
+				local closestDis, closestZone, pedcoords
+				if Config.DrawSprite then pedcoords = GetEntityCoords(PlayerPedId()) end
+				for k, zone in pairs(Zones) do
+					if Config.DrawSprite then
+						if #(pedcoords - zone.center) < Config.DrawDistance and not listSprite[k] then
```

Comment:
```suggestion
						if #(pedcoords - zone.center) < (zone.targetoptions.drawDistance or Config.DrawDistance) and not listSprite[k] then
```
You could change this to allow individual draw distance per zone